inject/src/test/groovy/io/micronaut/context/converters/StringToCharArraySpec.groovy

```diff
@@ -0,0 +1,20 @@
+package io.micronaut.context.converters
+
+import spock.lang.Specification
+
+class StringToCharArraySpec extends Specification {
+
+	def "test convert"() {
```

Comment:
The test should be injecting the conversion service and using that API to verify the conversion takes place